spec/cqm/models_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
+
+RSpec.describe Cqm::Models do
+  it "has a version number" do
+    expect(Cqm::Models::VERSION).not_to be nil
+  end
+
+  it "does something useful" do
+    expect(false).to eq(true)
```

Comment:
This seems...wrong, somehow.

Reply:
Yes, indeed it is. I'm going to add tests for each version of QDM that is included (so a test for each included modelinfo file), so basically some sanity checks to make sure the generator is handling each as expected. I'll ping you when finished.